inferentia/README.md

```diff
@@ -189,6 +209,10 @@ next available Neuron cores and not specific ones. See
 [Parallel Execution using NEURONCORE_GROUP_SIZES](https://awsdocs-neuron.readthedocs-hosted.com/en/latest/neuron-guide/appnotes/perf/parallel-ncgs.html?highlight=NEURONCORE_GROUP_SIZES)
 for more information.
 
+#### Batching with Tensorflow
+
+The batching flags have the same functionality for Tensorflow

```

Comment:
This is confusing. You should not have separate sections for PyTorch and TensorFlow. Add a common section after 'Serving Inferentia on Triton' section. You can call it `Using Triton's Dynamic Batching` with Inferentia models.